variables.tf

```diff
@@ -63,6 +93,24 @@ variable "public_ip" {
   description = "should ec2 instance have public ip?"
```

Comment:
consider changing this to `EC2` for consistency

Reply:
```suggestion
  description = "should EC2 instance have public ip?"
```